Makefile

```diff
@@ -15,7 +15,12 @@ dev-environment:
 docker-build:
 	docker build -t firewatch .
 
+docker-test:
```

Comment:
can you please add this at the top 
`IMAGE_BUILD_CMD = $(shell which podman 2>/dev/null || which docker)`

and use `IMAGE_BUILD_CMD` for build and run?

This will not force the user to install docker.